users/snowkuma/tap_dances.c

```diff
@@ -0,0 +1,104 @@
+#include "snowkuma.h"
+#include "process_records.h"
+
+// Tap Dance Definitions
+void kraken_safe_reset(qk_tap_dance_state_t *state, void *user_data) {
+	if (state->count >=3) {
+		// Reset the keyboard if you tap the key more than three times
+		reset_keyboard();
+		reset_tap_dance(state);
+	} else {
+    SEND_STRING(SS_LGUI(" "));
+    SEND_STRING("GitKraken.app\n");
+  }
+};
+
+void zoom_toggle(qk_tap_dance_state_t *state, void *user_data) {
+  if (state->count == 1) {
+    register_code(KC_LGUI);
+    register_code(KC_LSFT);
+    register_code(KC_A);
+    unregister_code(KC_A);
+    unregister_code(KC_LSFT);
+    unregister_code(KC_LGUI);
```

Comment:
Can replace all of this with just: 
```suggestion
    tap_code16(G(S(KC_A)));
```

Reply:
👍 That's cool I hadn't realised we could do that 😀